01_fib/nickolee/fibonacci_test.go

```diff
@@ -0,0 +1,53 @@
+package main
+
+import (
+	"bytes"
+	"reflect"
+	"strconv"
+	"testing"
+)
+
+func TestFibonacci(t *testing.T) {
+	var buf bytes.Buffer
+	out = &buf
+
+	main()
+
+	expected := strconv.Quote(`1
```

Comment:
any reason you're not using an interpreted string literal?

Reply:
`"1\n1\n2\n3\n5\n8\n13\n"`

Reply:
@Joshcarp, in wiki section, it is asked to [use raw strings](https://github.com/anz-bank/go-course/wiki/Lab-1:-Collected-Code-Review-Comments#consider-using-raw-strings)